README.md

```diff
@@ -1,2 +1,36 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+# Psuedocode for minigame word comparison section
+```
+# Snippet info: Checks similarity between a user-selected word and a preset secret word
+# Ignores characters beyond the length of the shorter word.
+# Todo: What should happen if user_word is longer than secret_word, but still equal up to that point?
+# Todo: Request clarification from contractor
```

Comment:
Good descriptive comments. Always sensible to document edge cases and possible error conditions.